moshi/src/main/java/com/squareup/moshi/Types.java

```diff
@@ -49,7 +53,7 @@ private Types() {
    * Returns the subset of {@code annotations} without {@code jsonQualifier}, or null if {@code
    * annotations} does not contain {@code jsonQualifier}.
    */
-  @CheckReturnValue public static @Nullable Set<? extends Annotation> nextAnnotations(
+  public static @Nullable Set<? extends Annotation> nextAnnotations(
```

Comment:
Why drop @CheckReturnValue?

Reply:
Just moved it to the Types type. That applies it to all the methods.